From 23126e956266c72a45228fafc7b5fa1a73840638 Mon Sep 17 00:00:00 2001
From: Michal Hrusecky <michal@hrusecky.net>
Date: Mon, 29 Jun 2020 18:30:46 +0200
Subject: [PATCH] netdata: Disable access from everywhere and compression

To be able to proxy nettdata through lighttpd and to require some sort
of authetification, changing the defaults to forbid remote access from
every IPv4 private segment possible by default (security enhancement)
and disabling compression (to make proxy work).
---
 admin/netdata/files/netdata.conf | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/admin/netdata/files/netdata.conf b/admin/netdata/files/netdata.conf
index 46b87a8a7..a34754026 100644
--- a/admin/netdata/files/netdata.conf
+++ b/admin/netdata/files/netdata.conf
@@ -14,8 +14,10 @@
 	run as user = root
 
 [web]
-	allow connections from = localhost 10.* 192.168.* 172.16.* 172.17.* 172.18.* 172.19.* 172.20.* 172.21.* 172.22.* 172.23.* 172.24.* 172.25.* 172.26.* 172.27.* 172.28.* 172.29.* 172.30.* 172.31.*
-	allow dashboard from = localhost 10.* 192.168.* 172.16.* 172.17.* 172.18.* 172.19.* 172.20.* 172.21.* 172.22.* 172.23.* 172.24.* 172.25.* 172.26.* 172.27.* 172.28.* 172.29.* 172.30.* 172.31.*
+
+	allow connections from = localhost
+	allow dashboard from = localhost
+	enable gzip compression = no
 
 [plugins]
 	cgroups = no
-- 
2.27.0

